### DISCUSSION TITLE
Particles not responsive

### DISCUSSION THREAD

**Discussion by Igor-sam-svoj-rob** [matteobruni/tsparticles#3587] [Q&A] [answered]
I have a Navbar section in which I inserted tsParticles, and used for style (among others) 100% height. But I would like to change the height to 70% if the width of the screen is smaller then 768px... 

I must admit I have no idea how to change this  in the tsParticles...

Here is my code for the particles:

`function Navbar() {

  return (
    <>
      <Particles
        id="tsparticles"
        options={{
          fpsLimit: 120,
          fullScreen: {
            enable: true,
            zIndex: 1,
          },
          style: {
            position: "absolute",
            top: 0,
            left: 0,
            width: "100%",
            height: "100%",
          },
          interactivity: {
            events: {
              onClick: {
                enable: true,
                mode: "push",
              },
              onHover: {
                enable: true,
                mode: "repulse",
              },
              resize: true,
            },
            modes: {
              push: {
                quantity: 1,
              },
              repulse: {
                distance: 100,
                duration: 0.6,
              },
            },
          },
          particles: {
            color: {
              value: "##53b7e8",
            },
            links: {
              color: "#53b7e8",
              distance: 150,
              enable: true,
              opacity: 0.5,
              width: 1,
            },
            collisions: {
              enable: true,
            },
            move: {
              direction: "none",
              enable: true,
              outMode: "bounce",
              random: true,
              speed: 2,
              straight: false,
            },
            number: {
              density: {
                enable: true,
                area: 800,
              },
              value: 40,
            },
          },
          detectRetina: true,
        }}
      />`

**Comment by matteobruni** [comment, marked as answer]
> I have a Navbar section in which I inserted tsParticles, and used for style (among others) 100% height. But I would like to change the height to 70% if the width of the screen is smaller then 768px...
> 
> I must admit I have no idea how to change this in the tsParticles...
> 
> Here is my code for the particles:
> 
> `function Navbar() {
> 
> return ( <> <Particles id="tsparticles" options={{ fpsLimit: 120, fullScreen: { enable: true, zIndex: 1, }, style: { position: "absolute", top: 0, left: 0, width: "100%", height: "100%", }, interactivity: { events: { onClick: { enable: true, mode: "push", }, onHover: { enable: true, mode: "repulse", }, resize: true, }, modes: { push: { quantity: 1, }, repulse: { distance: 100, duration: 0.6, }, }, }, particles: { color: { value: "##53b7e8", }, links: { color: "#53b7e8", distance: 150, enable: true, opacity: 0.5, width: 1, }, collisions: { enable: true, }, move: { direction: "none", enable: true, outMode: "bounce", random: true, speed: 2, straight: false, }, number: { density: { enable: true, area: 800, }, value: 40, }, }, detectRetina: true, }} />`

You can disable `fullScreen` and use your custom CSS to the canvas or the container

**Comment by Igor-sam-svoj-rob** [comment]
Solved it, tnx, setting the fullScreen to false did not solve it by itself, but I once I wrapped it inside the div which I changed with media queries, it worked like a charm.
